File: 26.06.2025/Multithreading Example.py
import threading

def print_numbers():
    for i in range(5):
        print(f"Number: {i}")

def print_letters():
    for letter in 'ABCDE':
        print(f"Letter: {letter}")

t1 = threading.Thread(target=print_numbers)
t2 = threading.Thread(target=print_letters)

t1.start()
t2.start()
t1.join()
t2.join()

File: 21.06.2025/Multithreading Example.java
class MyThread extends Thread {
    public void run() {
        for (int i = 0; i < 5; i++) System.out.println(i);
    }
}

public class ThreadDemo {
    public static void main(String[] args) {
        MyThread t = new MyThread();
        t.start();
    }
}
